pkg/api/handlers.go

```diff
@@ -0,0 +1,64 @@
+package api
+
+import (
+	"net/http"
+
+	"github.com/go-kit/kit/log/level"
+	"gopkg.in/yaml.v2"
+
+	"github.com/cortexproject/cortex/pkg/util"
+)
+
+// TODO: Update this content to be a template that is dynamic based on how Cortex is run.
+const indexPageContent = `
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="UTF-8">
+		<title>Cortex</title>
+	</head>
+	<body>
+		<h1>Cortex</h1>
+		<p>Admin Endpoints:</p>
```

Comment:
Currently endpoints are alphabetically sorted by URL. Maybe it's better to group the compactor and store-gateway endpoints at the end (since related to the experimental blocks storage).

Reply:
Makes sense, I'll put them at the end and add `(experimental blocks storage)`